### ISSUE TITLE
Broken link in README

### ISSUE THREAD

**Issue by nickynicolson** [plazi/ggxml2taxpub#70]
I'm directed to this link (https://github.com/plazi/eBioDiv/projects/1) to find more but it is a 404, presumably private (non-open)

**Comment by myrmoteras** [comment]
@nickynicolson yes, this is a closed management tool, but all there is is in the issues https://github.com/plazi/eBioDiv/issues

**Comment by nickynicolson** [comment]
> @nickynicolson yes, this is a closed management tool, but all there is is in the issues https://github.com/plazi/eBioDiv/issues

That's a closed resource too

**Comment by myrmoteras** [comment]
you are right. Bicikl, ebiodiv, are github repos we internally use to organize our work, and sometime discuss early stages of developments that will be moved to github issues, such as the matching service. If we would be very disciplined, no issue relating to a code github should reside in those repos above.
thanks for pointing this "mis-behavior" out.
